logging.conf

```diff
@@ -0,0 +1,40 @@
+[loggers]
+keys=root,data_transfer,dmpy.client
+
+[handlers]
+keys=console,tofile
+
+[formatters]
+keys=generic
+
+[logger_root]
+level=DEBUG
+handlers=console
+
+[logger_data_transfer]
+level=DEBUG
+handlers=console
+qualname=data_transfer
+propagate=0
+
+[logger_dmpy.client]
+level=DEBUG
+handlers=console,tofile
+qualname=dmpy.client
+propagate=0
+
+[handler_tofile]
+class=FileHandler
+level=DEBUG
+formatter=generic
+args=("./dtransfer.log",)
```

Comment:
This was primarily for testing -- @davidverweij do we want to output this this to a file? Note line 22, so only the dmpy output is output into this file and also the console. I'd assume for now we only want console logs as they would work best in docker, etc

Reply:
Yes, I'd say only console for now - and depending on how we are going to schedule the `data_transfer` runs, we could include a file (i.e. if we use a cronjob)

Reply:
Worth noting that console output will be picked up in docker contains for easy debugging, which we can pipe into a log outside of docker, etc